src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountDocumentServiceImpl.java

```diff
@@ -147,6 +149,14 @@ protected String getACHTransactionCode(String workdayAccountType) {
         }
     }
     
+    protected String determineStandardEntryClass(String achTransactionCode) {
+        if (StringUtils.isNotBlank(achTransactionCode)) {
+            return (StandardEntryClassValuesFinder.StandardEntryClass.valueOf((StringUtils.right(achTransactionCode, 3)).toUpperCase())).toString();
```

Comment:
This statement is quite complex and should be reworked a little more for clarity (and the same goes for the related statement in the PaymentWorks service class):

* Some parts have unnecessary wrapper parentheses. Those could be cleaned up as part of the other suggestions.
* To make the code's intent clearer, I would suggest splitting this line into at least two lines: One to derive the uppercased 3-character string and another to convert/validate it against the enum.
* For consistency with a KualiCo upgrade from several months earlier, any calls that uppercase or lowercase a string should include `Locale.US` as an argument.